src/wp-includes/class-wp-theme-json.php

```diff
@@ -604,19 +604,20 @@ class WP_Theme_JSON {
 	 * @var string[]
```

Comment:
This could have a `since` note:

```suggestion
	 * @since 6.9.0 Added `textInput` elements.
	 * @var string[]
```

Also, 6.1.0 added the `cite` element in addition to the others on [Core-56467](https://core.trac.wordpress.org/ticket/56467):
- r54105 for `heading`, `button`, and `caption`
- r54409 for `cite`

Reply:
good call!